spec/requests/api/person_escort_records_controller_show_spec.rb

```diff
@@ -19,6 +19,7 @@
     let(:person_escort_record_id) { person_escort_record.id }
 
     before do
+      person_escort_record.update_status_and_progress!
```

Comment:
since responses are added after creation, force recalculation of section progress